### DISCUSSION TITLE
FERMATAnot playing movies anymore on youtube

### DISCUSSION THREAD

**Discussion by ASIVMONET** [AndreyPavlenko/Fermata#285] [General]
Hello ! Just wondering if I'm the only one who has fermata auto but it no longer plays movies. It says get youtube app for that. Any fixes for this or not ? On a Samsung 22 ultra & 2021 nissan ! Please let me know . All help is appreciated!

**Comment by AndreyPavlenko** [comment]
It works for me.

**Comment by ASIVMONET** [comment]
Well it doesn't for me & I have the latest version. It says get the YouTube
app to play movies.  I can watch shows just not movies

**Reply by AndreyPavlenko** [reply]
Does it work in the Chrome web browser?

**Comment by ASIVMONET** [comment]
No, it does not. I get this message .

**Reply by AndreyPavlenko** [reply]
If it doesn't work in Chrome, then it wont work in Fermata either.

**Comment by ASIVMONET** [comment]
👍

Sent from my T-Mobile 5G Device
Get Outlook for Android<https://aka.ms/AAb9ysg>
________________________________
